src/mon/PGMap.cc

```diff
@@ -916,7 +916,13 @@ void PGMapDigest::dump_object_stat_sum(
   }
 
   uint64_t used_data_bytes = pool_stat.get_allocated_data_bytes(per_pool);
+  if (!per_pool) {
+    used_data_bytes *= raw_used_rate;
+  }
```

Comment:
@dongbula your commit message reads

> if `per_pool/per_pool_omap` is true，then `store_stats.allocated/store_stats.omap_allocated` will be used to get `used_bytes`, but they are not raw， so may be need to multiply `raw_used_rate`

but what you are changing in this PR is the case of where `per-pool` is false. is this expected?

Reply:
yes, in other cases, _get_allocated_data_bytes_ will return raw statics.
a possible error situation: https://tracker.ceph.com/issues/48385

Reply:
@dongbula i still don't get it. my question is more like

> what the issue you are fixing?

for instance, what is the symptom ? i am not able to connect https://tracker.ceph.com/issues/48385 with this fix without more context.

also the title of "fix a ceph df status bug" is not very specific. and the body of the commit message is almost empty. if this issue also impacts to one of the LTS branches. could you please create a tracker ticket, so we can backport the fix?